tabbycat/notifications/consumers.py

```diff
@@ -34,10 +34,10 @@ def _send(self, event, messages, records):
         try:
             mail.get_connection().send_messages(messages)
         except SMTPException as e:
-            self.send_error(e, _("Failed to send e-mails."), event)
+            self.send_error(e, _("Failed to send emails."), event)
             raise
         except ConnectionError as e:
-            self.send_error(e, _("Connection error sending e-mails."), event)
+            self.send_error(e, _("Connection error sending emails."), event)
```

Comment:
Just making a note, while trying to get this message to show up, I actually get
> 'NotificationQueueConsumer' object has no attribute 'send_error'
Because `send_error` isn't defined anywhere in this class or any of its parents.

I think this issue might go back as far as fa3f9e828b1dafdd3947c5d74843a5d46ab8cd19. The `send_error` method is defined in `tournaments.mixins.TournamentWebsocketMixin`.

Reply:
Opened #1547 to keep track of this.